packages/core/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "asynciterator": "^3.0.0",
-    "componentsjs": "^4.0.4",
+    "componentsjs": "4.5.0",
```

Comment:
Since 4.5.0 is in the range of ^4.0.4, this will probably not change anything. Or does it for you?
In any case, we'll want to keep using a semantic range for this instead of fixing to one specific version.